### ISSUE TITLE
Using --minify makes SVG render differently

### ISSUE THREAD

**Issue by loveleif** [gohugoio/hugo#8129]
### Description

SVG is rendered differently in some cases when using `--minify` because width and height is stripped.

### Steps to reproduce

1. Setup site, for example by following the [quick start guide](https://gohugo.io/getting-started/quick-start/).
2. Add the following html somewhere on the site:   
   
   ```
   <div style="width: 100%; height: 2rem;">
     <svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 46.113 46.113">
       <path d="M46.113 0L0 46.114h46.113z"/>
     </svg>
   </div>
   ```
3. Compare the rendered svg in `hugo server --minify` to `hugo server`. The svg will have a different size when minifiy is enabled because it strips the svg width and height.

### Screenshots

Following steps to reproduce I get the following result without minify:

<img width="473" alt="Screenshot 2021-01-10 at 15 20 20" src="https://user-images.githubusercontent.com/429963/104125533-a32fa180-5357-11eb-92de-2e9d4ba48e70.png">

And this result when using `--minify`:

<img width="643" alt="Screenshot 2021-01-10 at 15 20 00" src="https://user-images.githubusercontent.com/429963/104125535-a591fb80-5357-11eb-8d0a-bb159fa81e57.png">

### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
Hugo Static Site Generator v0.80.0/extended darwin/amd64 BuildDate: unknown
</pre>

### Does this issue reproduce with the latest release?

Yes.

**Comment by moorereason** [comment]
Please [file an issue upstream with `minify`](https://github.com/tdewolff/minify/issues), which Hugo uses for SVG minification.

Hugo is not using the latest release of minify, but I was able to reproduce the issue with the latest minify cli tool.  One thing I noticed in testing is that if you set width and height to "auto," minify retains the values.  See [the SVG spec](https://www.w3.org/TR/SVG2/geometry.html#Sizing) for what that means.  I didn't actually test how it renders in a browser.

**Comment by loveleif** [comment]
Filed issue upstream: https://github.com/tdewolff/minify/issues/367.

**Comment by loveleif** [comment]
Issue fixed in upstream https://github.com/tdewolff/minify/releases/tag/v2.9.11.

**Comment by jmooring** [comment]
Can no longer reproduce. Presume fixed in v0.81.0 with https://github.com/gohugoio/hugo/commit/66beac99c64b5e5fe7bec0bda437ba5858d49a36.
